pom.xml

```diff
@@ -74,9 +105,28 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>io.qameta.allure</groupId>
+                <artifactId>allure-maven</artifactId>
+                <version>2.10.0</version>
+            </plugin>
```

Comment:
This declaration is redundant because this plugin is not required outside of reporting scope (and is already defined there)

Reply:
Fixed

Reply:
Added this declaration back because without it there is no allure plugin in maven plugins list

Reply:
If you talk about Plugins list in Idea maven integration view
Run > Edit configurations > + > Maven
![image](https://user-images.githubusercontent.com/13015572/91314540-5770c700-e7bf-11ea-8376-8b588b547aa0.png)
![image](https://user-images.githubusercontent.com/13015572/91314624-6f484b00-e7bf-11ea-94bc-db2dbf607625.png)